boards/arm/nrf5340pdk_nrf5340/Kconfig

```
@@ -43,6 +43,12 @@ config BOARD_ENABLE_CPUNET
 	  may already have started the Network MCU.
 	default y if (BT || NET_L2_IEEE802154)
 
+config DOMAIN_NET_CORE_BOARD
+	string
+	default "nrf5340pdk_nrf5340_cpunet"
+	help
+	  The name of the BOARD corresponding to the 'NET_CORE' domain.
```

Not sure this is clear to readers (at least it's a little cryptic to me).

How about something like (in case I understand the purpose correctly)
```suggestion
        The board which will be used for NET_CORE domain when creating a multi image application where one or more images should be located on another board. For example hci_rpmsg on the nRF5340_cpunet for Bluetooth applications.
```